spec/view_component/form/date_select_component_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+RSpec.describe ViewComponent::Form::DateSelectComponent, type: :component do
+  let(:object)  { OpenStruct.new }
+  let(:form)    { form_with(object) }
+  let(:options) { {} }
+  let(:html_options) { {} }
+
+  let(:component) { render_inline(described_class.new(form, object_name, :birth_date, options, html_options)) }
+  let(:component_html_attributes) { component.css("select").first.attributes }
+
+  context "with simple args" do
+    it do
+      expect(component.to_html).to start_with(
```

Comment:
What do you think about testing only the first line with `start_with` ? Else the rendered component is very long and depends of the current date (which is `selected`).

Reply:
Let's start like this then.